server/api/team.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import Router from 'koa-router';
+import httpErrors from 'http-errors';
+
+import User from '../models/User';
+import Team from '../models/Team';
+
+import auth from './middlewares/authentication';
+import pagination from './middlewares/pagination';
+import { presentUser } from '../presenters';
+
+const router = new Router();
+router.use(auth({ adminOnly: true }));
+
+router.post('team.users', pagination(), async ctx => {
```

Comment:
This endpoint definitely seems like it should be open to more than admins. We'll need to list the folks in a team for everyone - perhaps exposing the details should just be admin only?